producer.py

```diff
@@ -46,14 +43,23 @@
 from thoth.python import AIOSource
 from thoth.python.exceptions import NotFoundError
 
+try:
```

Comment:
The reason the `__version__` was kept in the version.py 
is as we use kebechet for release, it currently only recognizes these list of files.
https://github.com/thoth-station/kebechet/blob/4c52e72a0ee0b9299b5165d77cc50c33b58f0350/kebechet/managers/version/release_triggers.py#L104

Reply:
Hum.
Relevant:
- https://github.com/thoth-station/kebechet/issues/1062 (probably)
-  https://github.com/thoth-station/core/issues/360

Do you mean kebechet require handling version in those way to handle releases ? Should'nt that be separate ?
(I don't see support for pyproject.toml, setup.cfg / others here. Would that mean that python packages using those can't use kebechet for that purpose ?)

Reply:
yes, correct, there isn't support for these in kebechet 
thanks for the issue, let's make kebechet more better. :+1: